tests/VolleyManagement.UnitTests/Services/GameReportService/Comparers/PivotStandingsComparer.cs

```diff
@@ -6,6 +6,39 @@
 
     internal class PivotStandingsComparer : IComparer<PivotStandingsDto>
     {
+        private TeamStandingsDtoComparer teamsComparer;
+        private bool hasComparerByResult;
+
+        public PivotStandingsComparer()
+        {
+            this.teamsComparer = new TeamStandingsDtoComparer();
+            hasComparerByResult = true;
+        }
+        public void WithPointsComparer()
+        {
+            teamsComparer.HasComparerBySets = false;
```

Comment:
Well, you indicated that issue yourself. Yes, there is duplication and it should be fixed.